kong/kong_test.go

```diff
@@ -113,3 +121,22 @@ func runWhenKong(t *testing.T, semverRange string) {
 	}
 
 }
+
+// runWhenEnterprise skips a test if the version
+// of Kong running is not enterprise edition.
+func runWhenEnterprise(t *testing.T) {
```

Comment:
This function should take a semver range as well. There are new endpoints introduced in newer versions of enterprise product that won't be supported by old versions and that will lead to test breakage.

Reply:
I've added support for semver range. thanks